### PR TITLE
introducing make targets for kafka and setting evn_vars

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,26 +42,30 @@ test-unit:
 
 # Run only E2E tests from the current repo.
 test-e2e:
-	UNINSTALL_STRIMZI="false" ./hack/strimzi.sh
```
Comment (matzew) [inline Makefile:45]: @aliok let's keep current e2e "small"

File: Makefile [hunk 2]
```diff
@@ -42,26 +42,30 @@ test-unit:
...
 	UNINSTALL_STRIMZI="false" ./hack/strimzi.sh
-	TEST_KNATIVE_E2E=true TEST_KNATIVE_UPGRADE=false ./test/upstream-e2e-tests.sh
+	TEST_KNATIVE_KAFKA=true TEST_KNATIVE_E2E=true TEST_KNATIVE_UPGRADE=false ./test/upstream-e2e-tests.sh
 
 # Run only upstream upgrade tests.
 test-upstream-upgrade:
 	UNINSTALL_STRIMZI="false" ./hack/strimzi.sh
-	TEST_KNATIVE_E2E=false TEST_KNATIVE_UPGRADE=true ./test/upstream-e2e-tests.sh
+	TEST_KNATIVE_KAFKA=true TEST_KNATIVE_E2E=false TEST_KNATIVE_UPGRADE=true ./test/upstream-e2e-tests.sh
+
```
Comment (matzew) [inline Makefile:64]: @aliok for all the upstream things, we run the contrib/kafka tests w (and enabled them via the VAR)

File: Makefile [hunk 3]
```diff
@@ -42,26 +42,30 @@ test-unit:
...
+# Run only E2E tests from the current repo.
+test-e2e-kafka:
```
Comment (matzew) [inline Makefile:66]: Now target for `e2e kafka`....  will be added to the CI job ...

File: test/e2e-tests.sh [hunk 4]
```diff
@@ -27,6 +27,10 @@ failed=0
 (( !failed )) && downstream_serving_e2e_tests || failed=4
 (( !failed )) && downstream_eventing_e2e_tests || failed=5
 
+if [[ $TEST_KNATIVE_KAFKA == true ]]; then
+  (( !failed )) && downstream_knative_kafka_e2e_tests || failed=6
```
Comment (matzew) [inline test/e2e-tests.sh:31]: @aliok let's separate the kafka e2e?

File: test/lib.bash [hunk 5]
```diff
@@ -93,6 +93,9 @@ function downstream_serving_e2e_tests {
   return $failed
 }
 
+function downstream_knative_kafka_e2e_tests {
+}
```
Comment (matzew) [inline test/lib.bash:97]: empty func, but I want to relocate the kafka e2e file / or run them separatedly

File: Makefile (hunk 3 above)
Comment (aliok) [reply Makefile:66]: +1

File: test/e2e-tests.sh (hunk 4 above)
Comment (aliok) [reply test/e2e-tests.sh:31]: sounds good.

**Pull request merged**